EndToEndGrpcTest(final)

yajurobii opened the pull request:
This test starts datastore gRPC server with the datastore implementation. It then connects a channel to it and wraps it with the GrpcDataStoreComputeAPI. A compute gRPC server with the computecontrollerapiimplementation and the user compute implementation. Build a UserSubmission just like client. Calls .submit over gRPC. Assertions to ensure pipline is working as intended. This accomplishes thorough testing over the gRPC from end to end, verifying behavior works as intended.

yajurobii committed:
EndToEndGrpcTest(final)

This test starts datastore gRPC server with the datastore implementation. It then connects a channel to it and wraps it with the GrpcDataStoreComputeAPI. A compute gRPC server with the computecontrollerapiimplementation and the user compute implementation. Build a UserSubmission just like client. Calls .submit over gRPC. Assertions to ensure pipline is working as intended. This accomplishes thorough testing over the gRPC from end to end, verifying behavior works as intended.

poninka reviewed (approved):
Review done via Pair Programming

yajurobii committed:
Create GrpcLoadTest.java

This commit adds GrpcLoadTest, a version of the end to end test that: Starts servers on available ports, creates client stub to the compute server, submits 200 parallel user requests, varies the input values to test load, cache, and cache eviction, assertions to ensure success, and time measurement for execution of the process.
